tests/functional/flows/test_list_runs.py

```diff
@@ -74,3 +74,35 @@ def test_list_runs_paginated_response(run_line, limit_delta):
             uuid.UUID(row[0])
         except ValueError:
             pytest.fail(f"Run ID is not a valid UUID: {row[0]}")
+
+
+def test_list_runs_filter_role(run_line):
+    load_response("flows.list_runs").metadata
+
+    result = run_line(
+        [
+            "globus",
+            "flows",
+            "run",
+            "list",
+            "--filter-role",
+            "run_owner",
+            "--filter-role",
+            "flow_run_manager",
+        ]
+    )
+    output_lines = result.output.split("\n")
+    assert len(output_lines) == 4
+
+    header_line = output_lines[0]
+    header_row = [item.strip() for item in header_line.split(" | ")]
+    assert header_row == ["Run ID", "Flow Title", "Run Label", "Status"]
+
+
+def test_list_runs_invalid_filter_role(run_line):
+    load_response("flows.list_runs")
+
+    run_line(
+        "globus flows run list --filter-role up-up-down-down-left-right-left-right-b-a",
+        assert_exit_code=2,
+    )
```

Comment:
It looks like these tests might be missing some assertions?

Reply:
Specifically:
- the one above only asserts the header row
- this one should probably check that the usage error message is what we expect

Reply:
The above only asserts the header row b/c I felt the other tests did a good job testing the table output. I don't mind adding that.

I can add an error message assertion to the invalid filter role test, good call. I normally do add expected error messages to my tests (not sure why i didn't this time).